components/ModEditSubmissionTopbar.vue

```diff
@@ -27,11 +26,11 @@
 import SVGCopyContent from '~/assets/icons/content-copy.svg'
 import SVGSuccessCheckMark from '~/assets/icons/checkmark-square.svg'
 import {  ref, type Ref } from 'vue'
-import { useModerationScreenStore } from '~/stores/moderationScreenStore'
+import { useModerationSubmissionsStore } from '#imports'
```

Comment:
let's use  the explicit `~/stores/moderationScreenStore`

Reply:
I will of course do this. But what is the benefit out of curiosity? Readability or is there another reason?